wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/ShareableDebugReportView.swift

```diff
@@ -0,0 +1,140 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+import WireDataModel
+import WireDesign
+
+class ShareableDebugReportView: UIView {
+
+    // MARK: - Constants
+
+    private enum LayoutConstants {
+        static let spacing: CGFloat = 2
+        static let padding: CGFloat = 12
+        static let iconSize: CGFloat = 28
+    }
+
+    // MARK: - Views
+
+    private lazy var topLabel: DynamicFontLabel = {
```

Comment:
In the case of a UIView subclass (vs a UIViewController) I don't think it is worth making its properties `lazy` as they will be created in the init anyway.

Reply:
https://github.com/wireapp/wire-ios/pull/1658/commits/261c26dd45fddd4f82a35ad6c5419f1c0d19acb7